src/styles/blocks/catalog.scss

```diff
@@ -1,25 +1,141 @@
+p,
+h2 {
+  margin: 0;
+}
+
 .catalog {
-  display: flex;
+  display: grid;
   flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  gap: 48px;
+  justify-content: center;
   padding: 50px 0;
 
-  &__card {
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $black;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    transition: color 300ms;
+  }
+
+  &:hover &__title {
+    color: $text-color-hover;
+  }
+
+  &__code {
+    width: 100%;
+    color: $grey;
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__rating {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding: 16px 0 24px;
+  }
+
+  &__reviews {
+    color: $black;
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: $grey;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__price-value {
+    color: $black;
+    line-height: 18px;
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__link {
     display: flex;
     justify-content: center;
     align-items: center;
-    box-sizing: border-box;
-
-    width: 200px;
-    height: 300px;
-    border: 1px solid #ccc;
+    width: 100%;
+    height: 40px;
+    background-color: $blue;
     border-radius: 5px;
-    margin-bottom: 48px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    border: 1px solid $blue;
+    transition: all 300ms;
```

Comment:
Why all? It's really bad for perfomance. Better specify the properties
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

Reply:
Hey, @ivanm2706!
What about this comment?